src/features/island/plots/CropReward.tsx

```diff
@@ -2,7 +2,7 @@ import { Panel } from "components/ui/Panel";
 import React, { useContext, useEffect, useRef, useState } from "react";
```

Comment:
This component can now be removed right? We will just use the `ChestReward`

Reply:
Done!